### PR TITLE
Node: Fix nvm ownership

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Adds correct ownership to `nvm` group for `$NVM_DIR`

### CODE REVIEW

**Comment by johnnyreilly** [comment]
oh is this going to resolve the weird permissioning issues I've been getting with Node I wonder?

File: src/node/install.sh [hunk 1]
```diff
@@ -107,7 +107,7 @@ export NVM_SYMLINK_CURRENT=true
 if [ -d "${NVM_DIR}" ]; then
     echo "NVM already installed."
     if [ "${NODE_VERSION}" != "" ]; then
-       su ${USERNAME} -c ". $NVM_DIR/nvm.sh && nvm install ${NODE_VERSION} && nvm clear-cache"
+       su ${USERNAME} -c "umask 0002 && . $NVM_DIR/nvm.sh && nvm install ${NODE_VERSION} && nvm clear-cache"
```
Comment (joshspicer) [inline src/node/install.sh:110]: What is the `2` bit doing here in `umask 0002`

File: src/node/install.sh [hunk 2]
```diff
@@ -185,4 +185,6 @@ if [ "${INSTALL_TOOLS_FOR_NODE_GYP}" = "true" ]; then
     fi
 fi
 
+find "${NVM_DIR}" -type d -print0 | xargs -n 1 -0 chmod g+s
```
Comment (joshspicer) [inline src/node/install.sh:188]: Why is the `chmod` on line 123 not enough?

File: src/node/install.sh (hunk 1 above)
Comment (samruddhikhandale) [reply src/node/install.sh]: Removed that, wasn't needed to fix the issue.

File: src/node/install.sh (hunk 2 above)
Comment (samruddhikhandale) [reply src/node/install.sh:188]: if you use a package manager and you are running as root, then the files/folders will not inherit those privs, for example. The `find` command helps address that.

This scenario fails when running tests in action

image

**Review by joshspicer — APPROVED** [review]

**Comment by samruddhikhandale** [comment]
@johnnyreilly The `node` Feature should now contain this change. 🚀 
Feel free to open an issue if it doesn't fix your permissions issue.

**Pull request merged**